src/app/State/cat.actions.ts

```diff
@@ -0,0 +1,19 @@
+import { createAction, props } from '@ngrx/store';
+import { IBreed } from '../shared/interfaces';
+
+export const getBreeds = createAction('[Cats] Get Breeds');
+export const getBreedsSuccess = createAction(
+  '[Cats] Get Breeds Success',
+  (breeds: IBreed[]) => ({ breeds })

```

Comment:
why we cannot set props as { breeds } same we do further? Let's have similar approach for all